samples/core/src/main/java/io/oasp/gastronomy/restaurant/offermanagement/dataaccess/api/ProductEntity.java

```diff
@@ -11,6 +9,8 @@
 
 import org.hibernate.envers.Audited;
 
+import io.oasp.gastronomy.restaurant.offermanagement.common.api.Product;
```

Comment:
why has this line moved? we should be working with the same formatting settings

was wrong on the first place?

if not, this file should not be included... as the change seems unnecesary